<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Train Monitor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnOne">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdWheelRotationCount">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdTimeDiffWheelTurn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdTimeCount">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdSpeed">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>371</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdPwm">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdGyroX">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>120</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdGyroY">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>230</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdGyroZ">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>350</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wheel Rotation Count</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time difference per wheel turn</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Milliseconds since start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Motor PWM Power</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>100</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gyro X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>210</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gyro Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>330</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gyro Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>823</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
